src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -555,15 +556,27 @@ public function prependExtensionConfig($name, array $config)
      *  * The parameter bag is frozen;
      *  * Extension loading is disabled.
      */
-    public function compile()
+    public function compile(/* $resolveEnvPlaceholders = false */)
```

Comment:
IMHO, we shouldn't plan for additional unplanned features :) This can be changed easily in the future *if* needed.